tests/integration/_common.py

```diff
@@ -233,6 +239,21 @@ def fixture_path_from_request(request, index):
     return dir_path, file_name
 
 
+def container_runtime_or_fail():
+    """find a container runtime, prefer podman
+    fail if neither available"""
+    # pylint: disable=import-outside-toplevel
+    import subprocess
+
+    for runtime in ("docker", "podman"):
```

Comment:
The doc comment says "prefer podman", but this tries docker first and returns if it finds it.

Reply:
Also, should this list come from somewhere else, so there's only one place to update if we add support for another container runtime in the future? We hardcode a similar list in `cli_args`.